ops/pebble.py

```diff
@@ -526,7 +583,7 @@ def wait_change(
 
             time.sleep(delay)
 
-        raise TimeoutError(
+        raise PollTimeout(
```

Comment:
The former type name seems better. The new name assumes a particular implementation of waiting, while the former did not. It was also nice to have a common suffix, but that's arguably subjective.

Reply:
Makes sense -- I'll revert this name.